experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/editableTree.events.spec.ts

```diff
@@ -10,6 +10,11 @@ import { FieldKinds } from "../../../feature-libraries";
 import { ForestType, SharedTreeFactory } from "../../../shared-tree";
 import { typeboxValidator } from "../../../external-utilities";
 import { AllowedUpdateType, SchemaBuilder, leaf } from "../../..";
+import { viewWithContent } from "../../utils";
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
Not for this PR, but general question: should we just disable this rule for tests? Having to add these comments everywhere is cumbersome, and it seems to be a pattern we're fine with in tests.

Reply:
Yea, maybe. Or we can put test files alongside the things they're testing rather than in a separate test folder. There were objection to that - I forget what they were.